### PR TITLE
RELEASE: 9/30, Ansible Automation Platform integration doc

### PR DESCRIPTION
**Pull request by akristen**
Updating yml file to put doc in its permanent home. Need a review that addresses:

- whether the flow of content makes sense
- good areas to incorporate the “value” of the feature.
- whether the actual structure of the page is attractive + usable

tl;dr of this doc is: it’s a dashboard. There's nothing specific to Ansible Tower, but instead builds on existing technologies to get data into New Relic (it builds on the infra agent and our Prometheus remote write integration). It's a good solution to people who have automated IT ~things and they want to view *all* of their data in a one stop shop (rather than looking at different logs and so on).

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+You would use this query:
+
+```sql
+FROM Metric SELECT count(status) FACET status SINCE 7 days ago WHERE prometheus_server = 'prometheus_remote' AND job = 'tower'
+```
+</Collapser>
+</CollapserGroup>
+
+Now you can view your data with your new Ansible Tower dashboard. After navigating to New Relic, find the **Dashboards** tile in the top nav.
```
Comment (rhetoric101) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx:133]: @akristen this seems like it should appear first (is the cart before the horse?). If you've already asked users to create custom queries, they would do it from dashboards, no?

**Review by rhetoric101 — DISMISSED** [review]
@akristen this is a well-written set of instructions! I just inserted a few minor suggestions.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Our Ansible Tower integration builds atop our Prometheus remote write integration. To view your data in our quickstart dashboard, follow the below steps to integrate Ansible Tower, Prometheus, and the New Relic platform. 
+
+<CollapserGroup>
+<Collapser
+    id="logging"
+    title="Step 1. Connect Ansible Tower logs to the infrastructure agent"
+  >
+Navigate `/etc/newrelic-infra/logging.d/`. If you don't see a `logging.yml` file, then create one.
+
+Add this script to the `logging.yml` file:
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx:49]: Ah, okay! That's good to know. Thanks @rhetoric101

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
+---
+title: Get metrics from Ansible Tower
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure remote write
+  - Ansible Tower
+  - Ansible Tower quickstart
+metaDescription:
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx:9]: Hah! Too funny. Good catch.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+* Sets `"prometheus"` as the target so it can scrape data from itself.
+</Collapser>
+<Collapser
+  id="jobs"
+  title="Step 3. Grab Ansible Tower metrics"
+>
+
+With data from Prometheus exposed to New Relic, you can now integrate your Ansible Tower projects into the New Relic platform. 
+
+Add this script to that same  `prometheus.yml` file:
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx:95]: Have confirmed with @pkosanam that as long as it's inserted in the file, it does not matter where specifically.

**Comment by akristen** [comment]
We are currently rethinking what will scrape data from Ansible for New Relic, so am closing the PR since the content will change significantly

**Pull request closed without merging**